tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -277,3 +279,14 @@ def test_all_4_type_pvc_creation_deletion_scale(self, namespace, tmp_path):
             for k, v in fs_pvc_deletion_time.items():
                 csv_obj.writerow([k, v])
         logging.info(f"Delete data present in {log_path}-deletion-time.csv file")
+
+
+def get_pvc_count():
+    ocp_info = ocp.OCP().exec_oc_cmd("get nodes -o json")
+    count = 0
+    for node_chk in ocp_info["items"]:
+        if "node-role.kubernetes.io/worker" in node_chk["metadata"]["labels"]:
+            count += 1
+    pvc_count = count * 500
+    logging.info(f"Number of pvcs to be created = {pvc_count}")
```

Comment:
I think since you are using this number in the logging after each time you call this function, this line is 'over-logging'

Reply:
I think that this will get printed five times.  Once for the test_multiple_pvc_creation_deletion_scale test and one time each for the four test_all_4_type_pvc_creation_deletion_scale tests.  I am rerunning just to make sure.

Reply:
Verified that this gets printed five times.

Reply:
This information is also printed elsewhere in the run, so I will remove the logging.info statement